packages/server/src/db/linkedRows/LinkDocument.ts

```diff
@@ -17,6 +17,7 @@ import { LinkDocument } from "@budibase/types"
 class LinkDocumentImpl implements LinkDocument {
   _id: string
   type: string
+  tableId: string
```

Comment:
I'm a little uncertain about this name, because it's not actually a table ID. Could be confusing when reading the code to later find out that it's 2 conjoined table IDs.

Perhaps `lookupId`?